.env.example

```
@@ -101,10 +101,20 @@ TEST_PASSWORD=testpassword
+#
+# ZITADEL_ORG_ID is STRONGLY RECOMMENDED in shared Zitadel instances.
+# When set, OidcClient appends the `urn:zitadel:iam:org:id:<id>` scope
+# to every authorization request, forcing both login AND new-user
+# registration into that specific org. Without it, the hosted login
+# lands new passkey/email registrations in Zitadel's IAM-internal
+# default org, which carries no project roles — the user appears
+# "registered" but has no RBAC permissions and is invisible to
+# org-scoped admin APIs.
 ##
 ZITADEL_ISSUER=http://localhost:8080
 ZITADEL_CLIENT_ID=your-frontend-client-id
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ZITADEL_CLIENT_ID key should go before the ZITADEL_ISSUER key

(UnorderedKey)